pkg/collector/osmlogs_collector.go

```diff
@@ -67,21 +67,36 @@ func (collector *OsmLogsCollector) Collect() error {
 	if err != nil {
 		return err
 	}
-
+	resources := []string{"configmap", "serviceaccount", "service", "endpoint", "ingress"}
```

Comment:
It'd be great to make this into a multiline slice declaration - so each resource is visible:
```suggestion
	resources := []string{
	"configmap", 
	"serviceaccount", 
	"service", 
	"endpoint", 
	"ingress",
	}
```